var/spack/repos/builtin/packages/wannier90/package.py

```diff
@@ -61,10 +89,81 @@ def edit(self, spec, prefix):
         for key, value in substitutions.items():
             filter_file(key, value, self.makefile_name)
 
-    def install(self, spec, prefix):
+        if '+shared' in self.spec:
+            if self.spec.satisfies('@:2'):
```

Comment:
If you want more space, you can either combine these in a single if-statement:
```python
if spec.satisfies('@:2 +shared'):
```
or
```python
if '@:2 +shared' in spec:
```
or you can exit when this isn't needed:
```python
if not spec.satisfies('@:2 +shared'):
    return
```
That way the line length issue isn't as bad.

Reply:
I find the first two to be clearer as to the intention, but I'm a python noob...  Thanks for the suggestions!